portfolio/src/main/webapp/static/constants/projects.js

```diff
@@ -0,0 +1,33 @@
+export default (() => {
+    const projects = [
+        {
+            title: "Stone crusher management system",
+            description: "Tailored management/pos webapp for a Mexican based company dedicated to blast, smash and sell different stone types."
+        },
+        {
+            title: "E-mail phishing web crawler detection",
+            description: "Chrome extension that fetches incoming links in mail, tests them against malware and disables them if necessary. Increase links' trustfulness before accessing them."
+        }
+    ];
+
+    const projects_section = document.getElementById("projects");
+    projects.forEach(project => {
+        const title = document.createElement("span");
+        const description = document.createElement("p");
+
+        title.innerHTML = project.title;
+        title.classList.add("badge", "badge-secondary");
+        description.innerHTML = project.description;
+        
+        const project_to_append = document.createElement("li");
+
+        project_to_append.classList.add("list-group-item", "text-wrap");
+        project_to_append.style = "width: 100%;";
```

Comment:
It'd probably be best to consolidate all your styling to your .css files. That way in the future if someone was trying to adjust something, it's more transparent where the styling is coming from.

I'd recommend adding a class/id to your project_to_append element that your CSS file can use.

Reply:
That's a really good suggestion. That approach centralizes better the CSS styling.
I'll go ahead and resolve it in a new commit.